src/connector/mssql.rs

```diff
@@ -467,6 +487,16 @@ impl MssqlUrl {
             .transpose()?
             .unwrap_or(false);
 
+        let max_connection_lifetime = props
+            .remove("max_connection_lifetime")
+            .map(|param| param.parse().map(Duration::from_secs))
+            .transpose()?;
+
+        let max_idle_connection_lifetime = props
```

Comment:
Remember to remove this from the engines: https://github.com/prisma/prisma-engines/blob/master/query-engine/connectors/sql-query-connector/src/database/mssql.rs#L34